src/locales/locales.json

```diff
@@ -1 +1 @@
-["en-US", "cak", "cy", "de", "en-CA", "fr", "fy-NL", "hu", "it", "nl", "nn-NO", "pt-BR", "ro", "sv-SE", "vi", "zh-CN", "zh-TW"]
+["en-US", "cak", "cy", "de", "en-CA", "fr", "fy-NL", "hu", "id", "it", "kab", "nl", "nn-NO", "pt-BR", "ro", "sv-SE", "vi", "zh-CN", "zh-TW"]
```

Comment:
Nice! And since we're slowing down on releases, is now the time to consider adding the 90%+ translations too? Like "cs" and "el" and "sl"?

Reply:
I thought about it, but I don't know if there's any assurance they would get to 100%, and it can be very jarring to use an app that's _mostly_ translated

Reply:
Define "slowing down"? I would expect them to reach 100% at this point, since we're not adding new strings. But it would be nice to make sure we ship an update in a couple of weeks.